wandb/sdk/wandb_run.py

```diff
@@ -2658,24 +2662,28 @@ def _footer_file_pusher_status_info(
                 )
         else:
             raise ValueError(
-                f"got the type `{type(poll_exit_responses)}` for `poll_exit_responses`. expected either None, PollExitResponse or a Dict[str, Union[PollExitResponse, None]]"
+                f"Got the type `{type(poll_exit_responses)}` for `poll_exit_responses`. "
+                "Expected either None, PollExitResponse or a Dict[str, Union[PollExitResponse, None]]"
             )
 
     @staticmethod
     def _footer_single_run_file_pusher_status_info(
-        pool_exit_response: Optional[PollExitResponse] = None,
+        poll_exit_response: Optional[PollExitResponse] = None,
         *,
         printer: Union["PrinterTerm", "PrinterJupyter"],
     ) -> None:
         # todo: is this same as settings._offline?
-        if not pool_exit_response:
+        if not poll_exit_response:
             return
 
-        progress = pool_exit_response.pusher_stats
-        done = pool_exit_response.done
+        progress = poll_exit_response.pusher_stats
+        done = poll_exit_response.done
 
         megabyte = wandb.util.POW_2_BYTES[2][1]
-        line = f"{progress.uploaded_bytes/megabyte :.2f} MB of {progress.total_bytes/megabyte:.2f} MB uploaded ({progress.deduped_bytes/megabyte:.2f} MB deduped)\r"
+        line = (
+            f"{progress.uploaded_bytes / megabyte :.2f} MB of {progress.total_bytes / megabyte:.2f} MB uploaded "
+            f"({progress.deduped_bytes / megabyte:.2f} MB deduped)\r"
```

Comment:
```suggestion
            f"{progress.uploaded_bytes / megabyte :.3f} MB of {progress.total_bytes / megabyte:.3f} MB uploaded "
            f"({progress.deduped_bytes / megabyte:.3f} MB deduped)\r"
```

was meaning to do it and forgot... it shows zeros a lot of time, so was thinking of adding more precision